src/components/Products.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function Products ({category,  image, text, price}) {
+    return (
+        <article>
+            <span>{category}</span>
+            <img src={image} alt={text}/>
+            <p>{text}</p>
+            <h4>{price}</h4>
```

Comment:
Heel goed gedaan! Enige opmerking is dat er wordt geadviseerd om hier bij Products component hier de -, te zetten zodat je alleen een cijfer hoeft mee te geven :)